frontend/public/components/markdown-view.tsx

```diff
@@ -18,8 +18,17 @@ const markdownConvert = (markdown, extensions?: string[]) => {
     extensions,
   }).makeHtml(markdown);
 
-  return sanitizeHtml(unsafeHtml, {
-    allowedTags: [
+  // add hook to transform anchor tags
+  DOMPurify.addHook('beforeSanitizeElements', function(node) {
+    // nodeType 1 = element type
+    if (node.nodeType === 1 && node.nodeName.toLowerCase() === 'a') {
```

Comment:
Best avoid magic constants...
```suggestion
    if (node.nodeType === Node.ELEMENT_NODE && node.nodeName.toLowerCase() === 'a') {
```
Although, not sure if we need to check the node type 🤔 

Also, in the upstream example they provided on the DOMPurify readme, it appears the way to check a `tagName` would be via the second param? eg: [for "script" nodes](https://github.com/cure53/DOMPurify/blob/main/demos/hooks-mentaljs-demo.html#L33)